app-frontend/src/app/pages/library/scenes/detail/detail.html

```diff
@@ -1,16 +1,21 @@
 <div class="row content stack-sm">
-  <div class="column-8" ng-show="!$ctrl.loading">
+  <div class="column-8" ng-if="!$ctrl.loading">
     <div class="embedded-scrollable">
       <h1 class="h3 page-title">
         <a ui-sref="library.scenes.list">{{$ctrl.scene.id}}</a>
```

Comment:
It would be good to have a clearer way of navigating back to the scene list from the scene detail view. Clicking on the uuid seems weird to me.